### DISCUSSION TITLE
2021-06-07 (RC1)

### DISCUSSION THREAD

**Discussion by rtsisyk** [organicmaps/organicmaps#533] [Announcements]
- New organic logo
- New map data from June
- Update fonts
- [ios] Fix disappearing navigation bar
- [ios] Fix invalid height constraint in Place Page
- [ios] Fix invalid constraints in Download Map
- [ios] Remove old graphics from search results
- [android] Fix OSM editor for F-Droid build
- [android] Fix Android 6.0 support
- [android] Add KMZ/KML/KMB bookmarks importer
- [android] Fix TTS on Android 11+
- [android] Remove broken background track recorder
  Required by Google
- [android] Add an option to allow screen to sleep if required
  Contributed by Martin Woolley <martin@woolleynet.com>

Android:

- Google Play - https://play.google.com/store/apps/details?id=app.organicmaps
- Huawei AppGallery - https://appgallery.huawei.com/#/app/C104325611
- F-Droid - https://f-droid.org/packages/app.organicmaps/
- GitHub  - https://github.com/organicmaps/organicmaps/releases
```
11c45678c381a67694d9c1538ec783bb38bcaea05c5842186467d7330db4dab1  OrganicMaps-21060705-web-release.apk
```

iOS:

- TestFlight - https://testflight.apple.com/join/lrKCl08I

<hr /><em>This discussion was created from the release <a href='https://github.com/organicmaps/organicmaps/releases/tag/2021-06-07'>2021-06-07 (RC1)</a>.</em>

**Comment by tangsgod** [comment]
Thanx a lot for the work!
Is that normal that i had to uninstall the previous version to install this one ?
Also I had to re download the maps.
Nothing that disturbed me, but just to know :)
